Contribution-and-Creation/README.md

```diff
@@ -0,0 +1,47 @@
+# OSPO 101 Training Extension Module 6 & 7
+## Open Source Contribution and Creation
+
+This course series is an extension of modules 6 and 7 from the OSPO 101 training course and is based on the [content from the outbound open source guide](https://github.com/todogroup/outbound-oss). Before taking this course, we encourage students to go through the introductory courses on using and creating open source projects:
+
+* [OSPO 101 - Module 6: Understanding Upstream Open Source Projects](https://github.com/todogroup/ospo-career-path/blob/main/OSPO-101/module6/README.md)
+* [OSPO 101 - Module 7: Open Source Creation](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module7)
+
+***
+
+### Program Overview
+
+For people at organizations looking to enhance their open source and/or IT strategy (people shaping the process), delving into outbound open source – which involves contributing to existing projects or initiating new ones – can be a transformative experience. 
+This training course is designed to guide open source managers adn stakeholders within organizations through this journey.
+
+This program is designed around three main thematic areas, which will be formatted into modules:
+
+**MODULE 1 - COLLABORATION**
+
+* Transitioning from usage (also called consumption) to Contribution
+  * Diving Deep into starting your organization's journey in contributing to open source projects
+  * Exploring various contribution models and procedures suitable for enterprises of all scales
+  * Managing and encouraging spare-time contributions
```

Comment:
I would not talk about encouraging spare-time contributions. The goal of corporate outbound open source should be to make it possible and effective to contribute as part of the job. Spare-time contributions are something a company has to deal with but it should be up to the people doing them, if they want to do them or not, and it should never be part of a company strategy to rely on spare-time contributions.